packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -164,7 +174,15 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
         await this.deploy(options);
       }
       latch = 'open';
+
+      // Idle state, let customers know and start streaming function logs if requested
       this.emitWatching();
+      if (options.functionStreamingOptions?.enabled) {
```

Comment:
Seems like the FunctionLogStreamer should be given all of the streaming options in its ctor and then it can decide internally if `startStreamingLogs` and `stopStreamingLogs` should be a noop or not based on the given config. Then all the FileWatchingSandbox has to know about is calling start/stop streaming logs at the right times

Reply:
Sure, updated.